src/components/PeopleDropdown/PeopleDropdown.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useCallback, useRef, useState } from 'react';
+import { PeopleList } from '../PeopleList';
+import { Person } from '../../types/Person';
+
+function debounce(
+  // eslint-disable-next-line @typescript-eslint/ban-types
+  callback: Function,
+  delay: number,
+  // eslint-disable-next-line @typescript-eslint/ban-types
+  afterCallback: Function,
+) {
+  let timerId = 0;
+
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  return (...args: any) => {
+    window.clearTimeout(timerId);
+
+    timerId = window.setTimeout(() => {
+      callback(args[0]);
+      afterCallback(args[1]);
+    }, delay);
+  };
+}
+
+type Props = {
+  onQueryChange: (newQuery: string) => void,
+  onPersonSelected: (selectedPerson: Person) => void,
+  delay: number,
+  people: Person[],
+};
+
+export const PeopleDropdown: React.FC<Props> = React.memo(
+  ({
+    onQueryChange,
+    onPersonSelected,
+    delay,
+    people,
+  }) => {
+    const [isInputFocused, setIsInputFocused] = useState(false);
```

Comment:
The list should be closed if the input is not focused.
Look at the [example](https://mui.com/material-ui/react-autocomplete/#combo-box)
<img width="576" alt="Screenshot 2024-01-19 at 20 30 56" src="https://github.com/mate-academy/react_autocomplete/assets/26187792/49c60d15-e7a3-404a-b185-544b6ba252df">